lib/pr_checker.js

```diff
@@ -373,8 +374,8 @@ export default class PRChecker {
 
     // GitHub new Check API
     for (const { status, conclusion, app } of checkSuites.nodes) {
-      if (app && app.slug === 'dependabot') {
-        // Ignore Dependabot check suites. They are expected to show up
+      if (app && IGNORED_CHECK_SLUGS.includes(app.slug)) {
+        // Ignore app check suites. They are expected to show up
```

Comment:
```suggestion
        // Ignore GitHub Apps check suites. They are expected to show up
```
That makes me think; should we filter only status for checks from `{"slug":"github-actions"}`?

Reply:
Might be fine as long as we don't require checks from other apps :)

Reply:
I think just ignoring dependabot and codecov is fine for now, but if the project installs more apps, maybe then ignore all apps.